packages/coordinator/src/services/translation/request/dispense/dispense-notification.ts

```diff
@@ -29,8 +29,12 @@ import {
 import {auditDoseToTextIfEnabled} from "../dosage"
 import {isReference} from "../../../../utils/type-guards"
 import {OrganisationTypeCode} from "../../common/organizationTypeCode"
-import {CodingExtension} from "../../../../../../models/fhir"
-import {NonDispensingReason, PertinentInformation2NonDispensing} from "../../../../../../models/hl7-v3"
+import {Bundle, CodingExtension, MedicationDispense} from "../../../../../../models/fhir"
+import {
+  DispenseNotificationSupplyHeaderPertinentInformation1,
+  NonDispensingReason,
+  PertinentInformation2NonDispensing
+} from "../../../../../../models/hl7-v3"
```

Comment:
Consider importing only `hl7-v3` and using the `module.class` notation instead, e.g. `hl7v3.NonDispensingReason`, to reduce the number of imports/clutter.